main.tex

```diff
@@ -84,42 +84,39 @@ \section{Introduction}
 the responsibility for maintaining long-term reproducibility from the
 individual researcher to the broader community.
 
-
-Data sharing has taken on a variety of forms.  The most prominent are the major
-observatory archives: MAST, NOAO, ESO, IPAC, CADC, CDS (hosting Vizier and
-SIMBAD), NRAO, CXC, HEASARC, and ESA are the main
-organizations hosting raw and processed data from ground and space based
-telescopes.  These data archives also serve as the primary means for serving data
-to users when the data are taken in queue mode, i.e., when the data are taken
-while the observer is not on-site.
+Data sharing has taken on a variety of forms.  The most prominent are the
+major observatory archives: MAST, NOAO, ESO, ESA, IPAC, CDS, NRAO, CXC, HEASARC,
+and CADC are the main organizations hosting raw and processed data from
+ground and space based telescopes.  These data archives also serve as the
+primary means for serving data to users when the data are taken in queue
+mode, i.e., when the data are taken while the observer is not on-site.
 
 In addition to observatories and telescopes, individual surveys often share
 their full data sets.  In some cases, these data sets are shared via the
 observatory that acquired them, for example, the all-sky data acquired with
 Planck, WMAP, and COBE\@.  Other surveys, particularly ground-based surveys,
-serve their own data.  Examples include SDSS, 2MASS, UKIDSS, and likely many
-more in the near future.
+serve their own data.  Examples include SDSS, 2MASS, UKIDSS, VSA.
+
+%TODO: mention data collector sites, too, nasa\_exoplanet\_archive, simbad,
+%skyview, OAC, etc.
 
-Individual teams and small groups will often share their data via their own
-custom websites.  These services do not follow any particular standard and can
+Individual teams and small groups often share their data via their own
+custom websites, too.  These services do not follow any particular standard and can
```

Comment:
the `, too` is not needed.  I make an effort to remove extraneous 'also's, 'too's, and 'additionally's from my work because I tend to write them too often, and they don't add anything.